lib/revise_auth/model.rb

```diff
@@ -8,7 +8,7 @@ module Model
 
       validates :email, format: {with: URI::MailTo::EMAIL_REGEXP}, presence: true, uniqueness: true
       validates :unconfirmed_email, format: {with: URI::MailTo::EMAIL_REGEXP}, allow_blank: true
-      validates_length_of :password, minimum: 12
+      validates_length_of :password, minimum: 12, allow_nil: true
```

Comment:
What's the reason to allow nil?

Reply:
To make it possible to update the user model without providing the password. It is still required on create even with this.

Reply:
Ah right